docs/tasks/administer-cluster/pvc-protection.md

```diff
@@ -0,0 +1,144 @@
+---
+approvers:
+- msau42
+- jsafrane
+title: Persistent Volume Claim Protection
+---
+
+{% capture overview %}
+{% assign for_k8s_version="v1.9" %}{% include feature-state-alpha.md %}
+
+As of Kubernetes 1.9, persistent volume claims (PVCs) that are in active use by a pod can be protected from pre-mature removal.
+
+{% endcapture %}
+
+{% capture prerequisites %}
+
+- A v1.9 or higher Kubernetes must be installed.
+- As PVC Protection is a Kubernetes v1.9 alpha feature it must be enabled:
+1. [Admission controller](/docs/admin/admission-controllers/) must be started with the [PVC Protection plugin](/docs/admin/admission-controllers/#persistent-volume-claim-protection-alpha).
+2. All Kubernetes components must be started with the `PVCProtection` alpha features enabled.
```

Comment:
IIRC, the feature is about apiserver, controller-manager and kubelet. Enabling it on scheduler or kube-proxy components are not needed.

Reply:
Assuming [the approved scheduler changes](https://github.com/kubernetes/kubernetes/pull/55957) won't make it into K8s 1.9 it won't be necessary to enable it on scheduler and kube-proxy components.

If [the scheduler changes](https://github.com/kubernetes/kubernetes/pull/55957) had made it into K8s 1.9 it would be necessary to enable it also on scheduler component.

Anyway, in beta version it will be necessary to enable it also on scheduler.

However, as @jsafrane suggested in [his comment](https://github.com/kubernetes/website/pull/6415#discussion_r153469358) it's better to suggest users to enable it on all components.

As the feature applies to majority of K8s components I also prefer to suggest users to enable it on all components.